### PR TITLE
Article Votes buttons and logic to those buttons added.Some styling c…

### PR DESCRIPTION
**Pull request by semih-suran**
…hanges made.

### CODE REVIEW

File: src/routes/ArticleCard.jsx [hunk 1]
```diff
@@ -25,10 +25,48 @@ const ArticleCard = ({ handleArticlesClick }) => {
...
+        }));
+        setTimeout(() => {
+          setArticle((prevArticle) => ({
+            ...prevArticle,
+            voteSuccess: false,
+          }));
+        }, 300);
+      })
+      .catch((error) => {
+        alert("<<< Failed to update the article vote >>>");
```
Comment (nc-mark-t) [inline src/routes/ArticleCard.jsx:51]: Just to note, when you use alert, it blocks the execution of any code so your app will freeze

File: src/routes/ArticleCard.jsx [hunk 2]
```diff
@@ -25,10 +25,48 @@ const ArticleCard = ({ handleArticlesClick }) => {
...
+
+    updateArticleVotes(article_id, voteValue)
+      // change voteValue to "nonsense" for the error feedback
+      .then(() => {
+        setArticle((prevArticle) => ({
+          ...prevArticle,
+          votes: prevArticle.votes,
+          voteSuccess: true,
+        }));
+        setTimeout(() => {
```
Comment (nc-mark-t) [inline src/routes/ArticleCard.jsx:43]: could this be handled without a set timeout, you could handle this based on the response

File: src/utils/api.js [hunk 3]
```diff
@@ -65,3 +65,16 @@ export const getCommentsByArticleId = (articleId) => {
       return err;
     });
 };
+
+export const updateArticleVotes = (articleId, voteValue) => {
+  return axios
+    .patch(`https://thenews-lhhv.onrender.com/api/articles/${articleId}`, {
```
Comment (nc-mark-t) [inline src/utils/api.js:71]: have a look at axios.create, it could be a nice refactor for this utils file

**Review by nc-mark-t — COMMENTED** [review]
Good work, nice functionality, it might be better to handle the resetting of the state without using set timeout

File: src/utils/api.js (hunk 3 above)
Comment (semih-suran) [reply src/utils/api.js:71]: thanks a million i ll dive into it ! 

https://blog.logrocket.com/understanding-axios-create/

**Pull request merged**